### PR TITLE
CNF 5456 Revision of the networking stack pinning

### PR DESCRIPTION
**Pull request by kquinn1204**
[CNF-5456]:  Revision of the networking stack pinning

Version(s):
4.12 RN entry 
Issue:
https://issues.redhat.com/browse/CNF-5456

Link to docs preview:
https://53667--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#disabling-realtime-using-workload-hints-removes-receive-packet-steering-from-the-cluster

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -456,18 +456,27 @@ Enabling this service monitor eliminates the possibility that two queries sent a
...
+If you are upgrading to {product-title} {product-version} and have specified relative paths for additional Alertmanager secret keys that are referenced as files, you must change these relative paths to absolute paths in your Alertmanager configuration.
 Otherwise, alert receivers that use the files will fail to deliver notifications.
 ====
 
 
 [id="ocp-4-12-scalability-and-performance"]
 === Scalability and performance
 
+==== Disabling realtime using workload hints removes Receive Packet Steering from the cluster
+By default, a systemd service sets a Receive Packet Steering (RPS) mask for virtual network interfaces in the cluster. The RPS mask routes interrupt requests from virtual network interfaces according to the list of reserved CPUs defined in the performance profile. A crio hook script also sets an RPS mask for all virtual network devices at the container level.
```
Comment (JoeAldinger) [inline release_notes/ocp-4-12-release-notes.adoc:472]:
```suggestion
At the cluster level by default, a systemd service sets a Receive Packet Steering (RPS) mask for virtual network interfaces. The RPS mask routes interrupt requests from virtual network interfaces according to the list of reserved CPUs defined in the performance profile. At the container level, a crio hook script also sets an RPS mask for all virtual network devices.
```

Comment (JoeAldinger) [reply release_notes/ocp-4-12-release-notes.adoc:472]: I thought this might help clarify the two different levels RPS is being set at. Maybe it helps?

**Review by JoeAldinger — COMMENTED** [review]
/remove-label peer-review-in-progress
/label peer-review-done
/lgtm

I left one small comment that I thought might help with clarity.

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -456,18 +456,27 @@ Enabling this service monitor eliminates the possibility that two queries sent a
...
-If you are upgrading to {product-title} {product-version} and have specified relative paths for additional Alertmanager secret keys that are referenced as files, you must change these relative paths to absolute paths in your Alertmanager configuration. 
+If you are upgrading to {product-title} {product-version} and have specified relative paths for additional Alertmanager secret keys that are referenced as files, you must change these relative paths to absolute paths in your Alertmanager configuration.
 Otherwise, alert receivers that use the files will fail to deliver notifications.
 ====
 
 
 [id="ocp-4-12-scalability-and-performance"]
 === Scalability and performance
 
+==== Disabling realtime using workload hints removes Receive Packet Steering from the cluster
```
Comment (bergerhoffer) [inline release_notes/ocp-4-12-release-notes.adoc:471]: This needs an anchor above it

File: .s2i/bin/assemble [hunk 3]
```diff
@@ -1,71 +0,0 @@
-#!/bin/bash -e
```
Comment (bergerhoffer) [inline .s2i/bin/assemble:1]: @kquinn1204 This file deletion was in the other PR of yours that I just peer reviewed. I don't think it's supposed to be here. Can you remove it from this PR before we merge?

**Review by bergerhoffer — COMMENTED** [review]
I think there's an incorrect file deletion in here that should be addressed before merging. And also a missing anchor ID if you wouldn't mind fixing that too while you're in here

File: release_notes/ocp-4-12-release-notes.adoc (hunk 2 above)
Comment (kquinn1204) [reply release_notes/ocp-4-12-release-notes.adoc:471]: my bad I forgot the anchor tag. sorry

File: .s2i/bin/assemble (hunk 3 above)
Comment (bergerhoffer) [reply .s2i/bin/assemble:1]: Yeah I can see that your commit message changed to something about the .gitignore file. So if you do have to redo the commit, I'd recommend fixing the commit message here too.

Let me know if you need any help with removing that accidental file deletion that is showing up.

**Pull request closed without merging**